src/test/java/com/qa/restassured/NonStaticImports.java

```diff
@@ -0,0 +1,18 @@
+package com.qa.restassured;
+
+import io.restassured.RestAssured;
+
+import org.hamcrest.Matchers;
+
+public class NonStaticImports {
```

Comment:
Naming of Class is not meaningful. Always use a meaningful name of class.
If you want to name it as static or non static way of implementation, just give a comment for each of Test cases? 
That way you can easily identify your intention?

Reply:
Modify class name